code/datums/components/crafting/crafting.dm

```diff
@@ -198,6 +198,12 @@
 	var/list/contents = get_surroundings(crafter, recipe.blacklist)
 	var/send_feedback = 1
 	var/dest_turf = get_turf(crafter)
+
+	if(isopenturf(dest_turf))
+		var/turf/open/open_turf = dest_turf
+		if(!open_turf.allow_construction)
```

Comment:
is this just disallowing all crafting? We want to invalidate crafting ONTO the turf, not like, making razorwire spools in your hand